pallets/creditcoin/src/migrations/v8.rs

```diff
@@ -119,14 +110,12 @@ impl<T: Config> Migrate for Migration<T> {
 
 #[cfg(test)]
 mod tests {
-	use crate::helpers::extensions::IntoBounded;
 	use pallet_offchain_task_scheduler::tasks::TaskV2;
 
 	use super::*;
 	use crate::{
-		migrations::v6::OldUnverifiedCollectedCoins,
 		mock::{self, ExtBuilder, Test},
-		types::{test::create_unverified_transfer, ContractType},
+		types::{collect_coins::ContractType, test::create_unverified_transfer},
```

Comment:
contract type is related to collect coins?

Reply:
its defined in that crate, I didnt want to make too many changes, and since we still need some structs from collect coins for events, the crate is not deleted, so I just left the contract type inside there